From 5c09c2231403bf5afc18d5b09861df7491a46591 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Wed, 2 Sep 2020 15:08:46 -0700
Subject: [PATCH 14/15] pci: Remove unused error variables

This is already fixed for the upcoming -next refresh so no sending
upstream.

Fixes: 89a8561c63d3 ("PCI: Remove unused pcibios_pm_ops")
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 drivers/pci/pci-driver.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/pci/pci-driver.c b/drivers/pci/pci-driver.c
index e136f08ad92a..40be221e69cf 100644
--- a/drivers/pci/pci-driver.c
+++ b/drivers/pci/pci-driver.c
@@ -1034,7 +1034,6 @@ static int pci_pm_thaw_noirq(struct device *dev)
 {
 	struct pci_dev *pci_dev = to_pci_dev(dev);
 	const struct dev_pm_ops *pm = dev->driver ? dev->driver->pm : NULL;
-	int error;
 
 	/*
 	 * The pm->thaw_noirq() callback assumes the device has been
@@ -1166,7 +1165,6 @@ static int pci_pm_restore_noirq(struct device *dev)
 {
 	struct pci_dev *pci_dev = to_pci_dev(dev);
 	const struct dev_pm_ops *pm = dev->driver ? dev->driver->pm : NULL;
-	int error;
 
 	pci_pm_default_resume_early(pci_dev);
 	pci_fixup_device(pci_fixup_resume_early, pci_dev);
-- 
2.28.0

